publish_trips.py

```diff
@@ -113,12 +117,42 @@ def map_row(row):
         if key in FIELDS
     }
 
+def classify_bike_type(data):
+    """
+    Classifying Bikes into Electric or Classic types based on the ID pattern supplied by CapMetro.
+    All e-bikes will have 5 digit bike numbers starting with “15” and up.
+    """
+    for row in data:
+        if len(row["bicycle_id"]) == 5 and int(row["bicycle_id"][0:2]) >= 15:
```

Comment:
you might want to wrap this whole `if` block in a try/except. this will throw a `ValueError` if the bike id can't be parsed as integer.